SalesforceCommerceCloud/pwa-kit
# @W-14901147: Shopper SEO hook

# Description
This PR added query hooks for Shopper SEO APIs that has been added into `commerce-sdk-isomorphic` v1.12.0



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR
For testing purpose of this API, I've added redirects in [BM](https://zzrf-001.dx.commercecloud.salesforce.com/on/demandware.store/Sites-Site/default/ViewUrlRedirectList_52-Paging?csrf_token=qpJGKojMhFe1Gi9yhEc3OmEPqGx5g4HG5ahyoWZWOPHMouk74rX2t-953JQGdAKVX1hJ3yPRRt7HlMOV4YojLDSo1tr4H3D8E7rqmZVksmqNt1Ptj55etyO8IL5dCFc2uFAiTrmnrOpNs2L-Ye3curAPkV56iHpFuSaTfbpgbCm8Bu2GlI8=) that will used in the template Home page. 

Source: /sample-product 
Destination: /sample-product-redirected

- check out code
- npm ci at root
- `npm start` to start up Retail app
- Navigate to home page http://localhost:3000. Confirm that Shopper SEO useUrlMapping returns the data in the browser console
```
{
  copySourceParams: false,
  destinationUrl: '/sample-product-redirected',
  statusCode: '301'
}
```


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
